bin/znapzend

```diff
@@ -164,6 +164,15 @@ use 'pfexec' for zfs commands (e.g. when running znapzend as non-priviledged use
 
 use 'sudo' for zfs commands (e.g. when running znapzend as non-priviledged user)
 
+=item compression
+
+use 'compression' to add options -Lce to the zfs send command
+If a source and destination volume are both using compression, zfs send will, by
+default, uncompress the data before sending. Zfs recv will then compress it again before 
+writing it to disk. Using -c will skip the unnecessary uncompress-compress stages.
+This decreases CPU load on both source and destination as well as reduce network
+bandwidth. The -L and -e options were thrown in for good measure.
```

Comment:
maybe mention a key word for L and e in brackets ... for good measure is a bit very flippant I think